monitor-translations/http/rename-exclusion-query.json

```
@@ -0,0 +1,11 @@
+{
+  "agentType": "TELEGRAF",
+  "monitorType": "http",
+  "name": "rename-timeout",
```

following suggestion from other PR
```suggestion
  "name": "http-rename-timeout",
```